OpenRA.Mods.Common/Traits/RepairsBridges.cs

```diff
@@ -97,7 +97,6 @@ public void ResolveOrder(Actor self, Order order)
 
 				self.SetTargetLine(Target.FromOrder(self.World, order), Color.Yellow);
 
-				self.CancelActivity();
```

Comment:
This will break non-queued repair orders!

Put this behind a `!queued` check instead of removing it - look at how Infiltrates etc do it.